<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDepthDialog</class>
 <widget class="QDialog" name="ProfileDepthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile Depth</string>
  </property>
  <widget class="PaintWidget" name="drawing_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>1031</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="coords_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>398</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>d= z=</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintWidget</class>
   <extends>QWidget</extends>
   <header location="global">paint_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
